servicemanager/service/smplayservice.py

```diff
@@ -95,6 +98,10 @@ def start_from_binary(self):
         parent, _ = os.path.split(unzip_dir)
         force_pushdir(parent)
 
+        if "frontend" in self.service_data and self.service_data["frontend"]:
```

Comment:
it's no more hardcoded than this though is it? https://github.com/hmrc/service-manager/blob/master/servicemanager/smcontext.py#L214-L223

This just checks for a flag in a service's config `"frontend": true`. Other options could be:
- It could be based on service name but that seems even flakier
- We could check every service being started for `*.conf` files and grep them for an assets version. That doesn't seem very efficient though

Open to other suggestions